tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -27,6 +28,7 @@
 #include <unordered_map>
 #include <unordered_set>
 #include <utility>
+#include <valarray>
 #include <vector>
 

```

Comment:
No change requested, just an observation - I agree that it is unnecessary to add tests for the `complex` changes here, since the Spaceship paper is merely removing operators that C++20 can rewrite. Our existing test coverage for `complex` (in `libcxx` mostly but also `tr1`) should be sufficient.